utils/ofi_debug_ip.patch

```diff
@@ -23,14 +23,17 @@ index ee16d7e3a..1ee7d77a6 100644
 +	ofi_straddr(_buf, &_len, ofi_translate_addr_format(ofi_sa_family(_addr)), _addr);	\
 +	ofi_straddr(_src_buf, &_src_len, ofi_translate_addr_format(ofi_sa_family(_src_addr)),	\
 +		    _src_addr);									\
-+												\
-+	FI_WARN(&xnet_prov, FI_LOG_EP_CTRL, "error on %s From %s : %s\n", _buf, _src_buf,	\
-+		fi_strerror(err));								\
++	if (err != 0)										\
++		FI_WARN(&xnet_prov, FI_LOG_EP_CTRL, "error on %s From %s : %s\n",		\
++			_buf, _src_buf,	fi_strerror(err));					\
++	else											\
++		FI_INFO(&xnet_prov, FI_LOG_EP_CTRL, "error on %s From %s : %s\n",		\
```

Comment:
If err == 0, why do we print "error on"

Reply:
ah, sure, let me refine the message a bit.